### DISCUSSION TITLE
Horizontal field layout and validation issue

### DISCUSSION THREAD

**Discussion by veyronB2** [rjsf-team/react-jsonschema-form#4351] [Q&A]
Hi all,

I have create a custom template when I add fields horizontally. However, when there is an error, the fields get misaligned due to error message under one field. Is there a way around it? 

<img width="977" alt="image" src="https://github.com/user-attachments/assets/f9e90fcc-a4a1-4073-9131-8e1997ad472b">

**Comment by heath-freenome** [comment]
You will likely have to add custom CSS (via uiSchema [classNames](https://rjsf-team.github.io/react-jsonschema-form/docs/api-reference/uiSchema/#classnames) property) to deal with that.
